src/features/user/TreeMapper/Analytics/components/Export/index.tsx

```diff
@@ -43,13 +47,23 @@ export const Export = () => {
   const { t, ready } = useTranslation('treemapperAnalytics');
   const { projectList, project, fromDate, toDate } = useAnalytics();
   const { userLang } = useUserProps();
-  const { handleError } = useContext(ErrorHandlingContext);
+  const { setErrors } = useContext(ErrorHandlingContext);
 
   const [localProject, setLocalProject] = useState<Project | null>(null);
   const [localFromDate, setLocalFromDate] = useState<Date>(fromDate);
   const [localToDate, setLocalToDate] = useState<Date>(toDate);
   const [projectType, setProjectType] = useState<ProjectType | null>(null);
 
+  const { makeRequest } = useNextRequest<{ data: IExportData[] }>(
+    '/api/data-explorer/export',
+    HTTP_METHOD.POST,
+    {
+      projectId: localProject?.id,
+      startDate: localFromDate,
+      endDate: localToDate,
+    }
+  );
+
```

Comment:
Even though there isn't a scope of conflict here, adding an alias for `makeRequest` here would make the `handleExport` function a little more readable.

Reply:
@mohitb35 So the idea behind `useNextRequest` hook is to make a single component responsible of making a request thus, the function name `makeRequest`. Though there could be an exception where we can use it more than once which could result in conflicts so using alias there make sense but not in this case.